src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -1311,6 +1315,14 @@ configurationRegistry.registerConfiguration({
 			type: 'string',
 			default: '',
 			tags: ['notebookLayout']
+		},
+		// --- Start Positron ---
+		[NotebookSetting.workingDirectory]: {
+			markdownDescription: nls.localize('notebook.workingDirectory', "Default working directory for notebook kernels. Supports variables like `${workspaceFolder}`. When empty, uses the notebook file's directory. Any change to this setting will apply to future opened notebooks."),
```

Comment:
Could we link to all the options here? Or just list them all?
Am I correct in thinking these are the relevant ones?
`workspaceFolder, fileDirname, userHome, env:HOME, env:USERPROFILE`.